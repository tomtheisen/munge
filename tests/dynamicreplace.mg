def(r) get(target) => "_"
/(\w):(\w+)/ => {  
    $1 set(target) drop
    $2 do(r)
}