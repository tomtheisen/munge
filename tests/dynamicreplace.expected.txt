_bc_bc_bc
d_r_b_b_l_ty
foodbar